### DISCUSSION TITLE
Default variable All option should suggest a sane regexp.

### DISCUSSION THREAD

**Discussion by cyriltovena** [grafana/grafana#40123] [Dashboards, Panels & Visualizations]
It would be really helpful if the include all options suggested `(?s:.*)` or  `(?s:.+)` the problem is that most people will use naively `.*` and `.+` but in RE2 syntax for Prometheus by default this can't be optimized away because the `.` matches anything but newlines.

Not sure how we can make this in the UI, but a suggestion or advice would go a long way to improve user experience.

See https://github.com/prometheus/prometheus/issues/7859

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
